.github/workflows/test_build_deb.yaml

```diff
@@ -11,7 +11,7 @@ jobs:
     - name: setup apt
       run: apt update -y && apt upgrade -y
     - name: install debuild dependencies
-      run: apt install -y git devscripts config-package-dev debhelper-compat
+      run: apt install -y git devscripts config-package-dev debhelper-compat golang
```

Comment:
The [build instructions](https://cs.android.com/android/platform/superproject/+/master:device/google/cuttlefish/README.md;l=29;drc=9dba51e13a40b3b0eebbf3ec1ba7492b22452674) should probably be updated as well for this.

Reply:
Done: aosp/1938906